web/modules/custom/menu_rest_resource/src/Plugin/Rest/Resource/MenuResource.php

```diff
@@ -0,0 +1,188 @@
+<?php
```

Comment:
This file is identical to `web/modules/custom/menu_rest_resource/src/Plugin/rest/resource/MenuResource.php`